ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.tsx

```diff
@@ -1,16 +1,75 @@
+import { useEffect } from "react";
+
+import { useSelector, useDispatch } from "react-redux";
+
 import DiscoveriesListHeader from "../DiscoveriesListHeader";
 
 import Section from "app/base/components/Section";
 import { useWindowTitle } from "app/base/hooks";
+import { actions } from "app/store/discovery";
+import discoverySelectors from "app/store/discovery/selectors";
 
 const DiscoveriesList = (): JSX.Element => {
   useWindowTitle("Dashboard");
 
+  const discoveries = useSelector(discoverySelectors.all);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(actions.fetch());
+  }, [dispatch]);
+
   return (
     <Section
       header={<DiscoveriesListHeader />}
       headerClassName="u-no-padding--bottom"
-    />
+    >
+      <table>
+        <thead>
+          <tr>
+            <th style={{ width: "20%" }}>Name</th>
```

Comment:
We typically create a `_index.scss` file for each table, and assign each `th` and `td` in that column a class (e.g. `name-col`). This way we can write media queries for dropping columns at certain breakpoints.

You can see [MachineTestsTable](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/index.scss) does this. For now I wouldn't worry about using the `breakpoint-widths` mixin, just assigning static values will be enough and we can worry about the "responsiveness" later.